charts/datadog/values.yaml

```diff
@@ -1935,3 +1936,9 @@ providers:
   aks:
     # providers.aks.enabled -- Activate all specifities related to AKS configuration. Required as currently we cannot auto-detect AKS.
     enabled: false
+
+remoteConfiguration:
+  # remoteConfiguration.enabled -- Set to true to enable remote configuration on the cluster agent (if set) and the datadog agent.
```

Comment:
```suggestion
  # remoteConfiguration.enabled -- Set to true to enable remote configuration on the Cluster Agent (if set) and the node agent.
```

Reply:
Fixed in [bce36c7](https://github.com/DataDog/helm-charts/pull/1058/commits/bce36c7cb465befe6dd59927e6e6265fe91e5112)